STYLE.md

```diff
@@ -131,15 +131,188 @@ if status is not pebble.ServiceStatus.ACTIVE:
 
 ## Docs and docstrings
 
-### Use British English
+### Tones
+
+#### Avoid negative, be positive
+
+State conditions positively as what should happen, rather than what shouldn't. Focus on desired behaviour, and frame instructions and conditions around the expected or successful outcome, not failure cases.
+
+Example:
+
+- Avoid: "If the command doesn’t exit within 1 second, the start is considered successful." (Netagive)
+- Prefer: "If the command stays running for the 1-second window, the start is considered successful." (Positive)
+
+Exceptions: Only use negative phrasing (e.g., "if X does not happen") when:
```

Comment:
Elsewhere in this doc we say to avoid Latin abbreviations like "e.g." :-). I think we can just drop this parenthesis:
```suggestion
Exceptions: Only use negative phrasing when:
```

Please replace "e.g." with "for example" throughout.